Add support for Comments after Line Continuations new in VB 16. (#14247)
dotnet/docs: docs/visual-basic/getting-started/whats-new.md

## Current version

> Visual Basic 16 / Visual Studio 2019 Version 16.0  

Reviewer: ```suggestion
Visual Basic 16.0 / Visual Studio 2019 Version 16.0  
```

> For new features, see [Visual Basic 16](#visual-basic-16)

Reviewer: ```suggestion
For new features, see [Visual Basic 16.0](#visual-basic-160)
```

Visual Basic / Visual Studio .NET 2002
The first release of Visual Basic .NET

> ## Visual Basic 16

Reviewer: ```suggestion
## Visual Basic 16.0
```

Reviewer 2: Yes, it should be 16.0 in all cases

> 

Reviewer 2: ```suggestion
Visual Basic 16.0 focuses on supplying more of the features of the Visual Basic Runtime (microsoft.visualbasic.dll) to .NET Core and is the first version of Visual Basic focused on .NET Core. Many portions of the Visual Basic Runtime depend on WinForms and these will be added in a later version of Visual Basic. 

**Comments allowed in more places within statements**
```

@paul1956 Do you have a better header here?

Author: With some minor changes many WinForms applications with work, a reasonable set of instructions with necessary changes could be linked to. Also what was listed as 15.8 is in VB16 and should not be lost. The main thing missing is the My namespace and MsgBox listing it acknowledges what issues people might face.

> In Visual Basic 15. and Earlier comment are only allowed on blank lines before a statement, at the end of a statement or in specific places within a statement where an implicit line continuation is allowed.

Reviewer: ```suggestion
In Visual Basic 15.8 and earlier versions, comments are only allowed on blank lines, at the end of a statement, or in specific places within a statement where an implicit line continuation is allowed.
```

> 

Reviewer: There shouldn't be a paragraph break; this should be a single paragraph.

Author: Fixed

> Starting is Visual Basic 16 comments are also allowed after Explicit Line Continuation and within a statement on a line beginning with a space followed by an underscore.

Reviewer: ```suggestion
Starting with Visual Basic 16.0, comments are also allowed after explicit line continuations and within a statement on a line beginning with a space followed by an underscore.
```

> ```VB

Reviewer: ```suggestion

```vb
```
